src/pages/home/report/ReportActionsList.js

```diff
@@ -145,6 +141,10 @@ class ReportActionsList extends React.Component {
     }
 
     render() {
+        // Non-animating skeleton UI represents that chat history exists but is not loaded locally in Onyx.
+        // While offline if there is at least one non-pending action or if this is a new chat then we don't need to show the non-animating skeleton UI.
```

Comment:
How does being offline come into play here? This component does not seem aware of the network status anymore since we are removing `withNetwork`.

Reply:
Aaah, I was just trying to explain in comments and used that word. I will get rid of it.